articles/active-directory/develop/quickstart-v2-ios.md

```diff
@@ -114,6 +122,16 @@ In a terminal window, navigate to the folder with the downloaded code sample and
 >    ```swift
 >    let kClientID = "Enter_the_Application_Id_Here"
 >    ```
+> 1. If you're building an app for [AzureAD national clouds](https://docs.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints), replace the line starting with 'let kGraphEndpoint' and 'let kAuthority' with correct endpoints. For global access, use default values:
+>     ```objective-c
+>     let kGraphEndpoint = "https://graph.microsoft.com"
+>     let kAuthority = "https://login.microsoftonline.com/common"
+>     ```
+> 1. Other endpoints are documented [here](https://docs.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints). For example, to run the quickstart with AzureAD Germany, use following:
```

Comment:
Seems line 130 is still talking the same thing as line 125, will it be better not to create a new bullet point here? :)

Reply:
I can't seem to figure out how to remove the bullet without breaking enumeration and formatting :)